spec/factories.rb

```diff
@@ -1,4 +1,15 @@
 FactoryGirl.define do
+  trait :first_or_new_user do
+    before(:create) do |model, evaluator|
```

Comment:
Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.